packages/bot-web-ui/src/pages/bot-builder/quick-strategy/types.ts

```diff
@@ -129,3 +129,11 @@ export type TFormValues = Record<string, string | number | boolean> & {
     type?: string;
     duration?: string;
 };
+
+export type TBlocklyEvents = {
+    type: string;
+    element: string;
+    group: string;
+    oldValue: string;
+    blockId: string;
+};
```

Comment:
I think it should be in the shared TS folder. Not under QS 🙏

Reply:
Ok. Moving it.